CITATION.cff

```diff
@@ -5,6 +5,79 @@ identifiers:
 - type: doi
   value: 10.5281/zenodo.3555620
 authors:
+- given-names: Nicholas
+  family-names: Sofroniew
+  affiliation: Chan Zuckerberg Initiative
+  orcid: https://orcid.org/0000-0002-3426-0914
+  alias: sofroniewn
+- given-names: Talley
+  family-names: Lambert
+  affiliation: Harvard Medical School
+  orcid: https://orcid.org/0000-0002-2409-0181
+  alias: tlambert03
+- given-names: Grzegorz
+  family-names: Bokota
+  affiliation: University of Warsaw, Faculty of Mathematics, Informatics, and Mechanics
+  orcid: https://orcid.org/0000-0002-5470-1676
+  alias: Czaki
+- given-names: Juan
+  family-names: Nunez-Iglesias
+  affiliation: Monash eResearch Centre, Monash University
+  alias: jni
+- given-names: Peter
```

Comment:
@jni How could I possibly be in this slot if it's by commit?

Reply:
I added together this repo and docs. 😃

Reply:
@jni Did you do this manually, or you created a script to automatize this before each release?

Reply:
Manually. 😬

Reply:
Ah. 
In general, GitHub GUI counts also coauthor ship of commits, that is not easy accessible from API. 

@psobolewskiPhD it looks like @jni used data where being coauthors of PR (having accepted suggestion to merged PR) is counting, so your activity in reviewing PR and suggesting updates bum your position.